src/PermanentFilterProvider.tsx

```diff
@@ -32,16 +40,19 @@ export const PermanentFilterProvider = ({ children }: { children?: React.ReactNo
     mowasRegionalKeysReducer,
     []
   );
+  const [filterState, filterDispatch] = useReducer(overlayFilterReducer, []);
 
   const loadFilters = useCallback(async () => {
     const dataProviderIds = ((await readFromStore(DATA_PROVIDER_FILTER_KEY)) ?? []) as string[];
     const mowasRegionalKeys = ((await readFromStore(MOWAS_REGIONAL_KEYS)) ?? []) as string[];
+    const overlayFilter = ((await storageHelper.filter()) ?? []) as string[];
```

Comment:
don't start a new name with overlay...should be all the same starting with resource...

Reply:
name change implemented with this commit
https://github.com/smart-village-solutions/smart-village-app-app/pull/1188/commits/6832d3814e0fced2f8c3613d1e569788bdd5398a